src/index.js

```diff
@@ -0,0 +1,79 @@
+import './style.css';
+
+const todoList = [
+  {
+    description: 'Read the Bible',
+    completed: true,
+    index: 0,
+  },
+  {
+    description: 'Wash the car',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Prepare Breakfast',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Take a bath',
+    completed: false,
+    index: 3,
+  },
+  {
+    description: 'Play call of duty',
+    completed: false,
+    index: 4,
+  },
+  {
+    description: 'Practice coding challenges',
+    completed: true,
+    index: 5,
+  },
+  {
+    description: 'Go grocery shopping',
+    completed: true,
+    index: 6,
+  },
+];
+
+todoList.sort((a, b) => {
+  if (a.index < b.index) return -1;
+  if (a.index > b.index) return 1;
+  return 0;
+});
+
+const listSection = document.querySelector('#list-items');
+listSection.innerHTML = '';
+for (let i = 0; i < todoList.length; i += 1) {
+  const task = todoList[i];
+  let taskItem = `
+    <li class="d-flex s-between list-item">`;
+  if (task.completed) {
+    taskItem += `<span class="material-icons done" onclick="updateStatus(${task.index}, 'pending')">
```

Comment:
- [ ]  Whenever I click the check box, it throws a console error concerning the `onClick` event handler `updateStatus(${task.index}, 'pending')`. Kindly fix this error. :pray: 

![Screenshot from 2021-11-30 22-46-47](https://user-images.githubusercontent.com/64894927/144118436-ca9cd4dd-790c-4679-aaf6-d68445c0c434.png)

Reply:
Hello @henatan99 , Kindly note that the requested change is not part of the requirements. I was trying to implement an onlcick event that I will use in the next project activity as per requirement.

Reply:
However, I have fixed the issue.

Reply:
However, I have fixed the issue.